presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClient.java

```diff
@@ -245,6 +247,11 @@ protected String getTableSchemaName(ResultSet resultSet)
                     int scale = min(decimalDigits, getDecimalDefaultScale(session));
                     return Optional.of(decimalColumnMapping(createDecimalType(Decimals.MAX_PRECISION, scale), getDecimalRoundingMode(session)));
                 }
+                break;
+            case Types.BINARY:
+            case Types.VARBINARY:
+            case Types.LONGVARBINARY:
```

Comment:
You only tested one type, do we need tests for others?

Reply:
I've added testing for `BINARY` and LONGVARBINARY now. Let me know if you'd like them in the smoke tests too.